### ISSUE TITLE
generating the HTML pages...

### ISSUE THREAD

**Issue by mamund** [jam01/rest-discuss-archive#1]
i see the JSON files here but not the HTML renderings.

do you have plans to do that next?

**Comment by jam01** [comment]
Yeah, I'll peruse the forks and play around with the HTML converter

**Comment by mamund** [comment]
excellent. thanks for donating your time for a great cause.

**Comment by jam01** [comment]
Generated the yearly HTML pages using the same Archiver app. I'm afraid my HTML skills won't take it any further... labelling as help wanted

**Comment by mamund** [comment]
well - this is a good start.  

thanks again.
